packages/launch/src/focused-launch/summary/focused-launch-summary-item/index.tsx

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
If this component needs to be used outside of Focused Launch, we could look into making it a more generic "Launch" item, or even moving it to the `@automattic/onboarding` package

Reply:
Sure. I'll open a new issue and add it to the Focused Launch Backlog milestone